src/broadcast/rtc_streamer.ts

```diff
@@ -120,6 +120,9 @@ export class WebRTCStreamer extends Streamer {
         this.state = "ANSWER";
         break;
       case "ACTIVATED":
+        if (!this.isActive) {
+          this.onStateChange("GOING_LIVE"); // Starting show.
+        }
```

Comment:
You sure about this? `isActive` gets reset on L131, and I'm fairly sure shittyserver fires a DEACTIVATED when the news happens

Reply:
Gah, in which case, maybe I'd rather just make this manually reset from the nav menu